python/ray/rllib/dqn/dqn_evaluator.py

```diff
@@ -157,8 +157,9 @@ def _step(self, global_timestep):
         return ret
 
     def stats(self):
-        mean_100ep_reward = round(np.mean(self.episode_rewards[-101:-1]), 5)
-        mean_100ep_length = round(np.mean(self.episode_lengths[-101:-1]), 5)
+        n = self.config["smoothing_num_episodes"] + 1
```

Comment:
would be good to add a note that there's some weird thing with the last entry and we should fix